### DISCUSSION TITLE
Dynamic Clock

### DISCUSSION THREAD

**Discussion by xbloker** [zixpo/candybar#38] [Q&A] [answered]
Greetings! Can you please tell me how to make the Dynamic Clock work? The hands of the clock stand still :(

**Comment by INayra** [comment]


**Reply by xbloker** [reply]
Perhaps I asked a question in the wrong place, as here quite recently

**Comment by xbloker** [comment]
Solved my problem. She was close. Here is the solution in case someone is an anchor like me [https://gist.github.com/sarsamurmu/4619b125fe69f545723e0f87c96b5cbd](url)

**Comment by sarsamurmu** [comment, marked as answer]
Check out the "Making Dynamic Clock icon" section of this article - https://medium.com/@sarsamurmu/adaptive-icons-in-icon-pack-1efdc1f77545
